daisy_workflows/image_build/debian/build.py

```diff
@@ -62,32 +63,32 @@ def main():
     raise ValueError(
         'Metadata "google_cloud_repo" must be one of %s.' % REPOS)
 
-  utils.LogStatus('Bootstrap_vz manifest: %s' % bvz_manifest)
-  utils.LogStatus('Bootstrap_vz version: %s' % bvz_version)
-  utils.LogStatus('Google Cloud repo: %s' % repo)
+  logging.info('Bootstrap_vz manifest: %s' % bvz_manifest)
```

Comment:
These workflows use these message to report status (https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/daisy_workflows/image_build/debian/debian.wf.json#L61)... so all of these should be prefaced with "BuildStatus:" to make that work. Likewise, errors are "BuildFailed:".

Reply:
All the logging.info will be prefixed by "BuildStatus", but thought LogFormatter at  https://github.com/GoogleCloudPlatform/compute-image-tools/pull/474/commits/73f23f4124a399d190be292ca944adba86e422a5#diff-000c28015d35ee8c5a7977072608a5dcR345

Reply:
Oh awesome- sorry I missed that before :)